.github/workflows/codeql.yml

```diff
@@ -0,0 +1,66 @@
+name: "CodeQL vulnerability check"
+
+on:
+  push:
+    branches: ["main", "develop"]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: ["develop"]
+    types: [opened, synchronize, ready_for_review]
+
+jobs:
+  analyze:
+    name: Analyze
+    # Runner size impacts CodeQL analysis time. To learn more, please see:
+    #   - https://gh.io/recommended-hardware-resources-for-running-codeql
+    #   - https://gh.io/supported-runners-and-hardware-resources
+    #   - https://gh.io/using-larger-runners
+    # Consider using larger runners for possible analysis time improvements.
+    runs-on:
+      ${{ (matrix.language == 'swift' && 'macos-latest') || 'ubuntu-latest' }}
+    timeout-minutes: ${{ (matrix.language == 'swift' && 120) || 360 }}
```

Comment:
Swift?

Reply:
Yeah this was part of the code they suggest you use. I just forgot to take out this swift stuff. I think it'll be a while before we have the mlstacks ios interface...